examples/tomo/xray_trafo.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2014, 2015 The ODL development group
```

Comment:
Putting comment here (regarding the images).

Way to many images I'd say. What are they all for?